packages/validator/src/services/validatorStore.ts

```diff
@@ -562,13 +562,13 @@ export class ValidatorStore {
 
     const signingSlot = aggregate.data.slot;
     const domain = this.config.getDomain(signingSlot, DOMAIN_AGGREGATE_AND_PROOF);
-    const signingRoot =
-      this.config.getForkSeq(duty.slot) >= ForkSeq.electra
-        ? computeSigningRoot(ssz.electra.AggregateAndProof, aggregateAndProof, domain)
-        : computeSigningRoot(ssz.phase0.AggregateAndProof, aggregateAndProof, domain);
+    const isPostElectra = this.config.getForkSeq(duty.slot) >= ForkSeq.electra;
```

Comment:
nit: I think it's better to use `signingSlot` to determine `isPostElectra`. Just to be consistent with how `domain` is also determined by `signingSlot`.

Reply:
It looks like we always use the duty slot to determine the fork, in practice it shouldn't matter
https://github.com/ChainSafe/lodestar/blob/ad4ec7713d0ab39c54ea1ffa28d6f6f997769fa0/packages/validator/src/services/validatorStore.ts#L805
Could update this to use signing slot separately as it would touch unrelated parts to this PR